### PR TITLE
3P amp-ad adapter for Venatus Media

### PR DESCRIPTION
**Pull request by cwbeck**
Added support for Venatus Media in ads.

All tests passing. Let me know if I have missed anything.

### CODE REVIEW

**Comment by cwbeck** [comment]
I signed it!

**Comment by cwbeck** [comment]
I signed it!

I've committed from chrisb@venatusmedia.com (set as my email). I've signed the agreement as chrisb@venatusmedia.com for Venatus Media

File: ads/venatus.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+import {writeScript, validateData} from '../3p/3p';
+
+/**
+ * @param {!Window} global
+ * @param {!Object} data
+ */
+export function venatus(global, data) {
+  validateData(data, ['sid', 'pid'], []);
+  const url = 'https://api.venatusmedia.com/deliver/tag/' + data.sid + '/' + data.pid;
```
Comment (alanorozco) [inline ads/venatus.js]: Line is over 80 chars. Please use string interpolation and split assignment in two lines:

```javascript
  const url =
      `https://api.venatusmedia.com/deliver/tag/${data.sid}/${data.pid}`;
```

File: ads/venatus.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+</amp-ad>
+```
+
+## Configuration
+
+For configuration details and to generate your tags, please contact us :- https://www.venatusmedia.com/contact or chrisb@venatusmedia.com
+
+Required parameter:
+
+- data-sid
```
Comment (alanorozco) [inline ads/venatus.md]: Please specify what these parameters are for.

File: ads/venatus.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+	data-sid="123456789"
+	data-pid="987654321">
+</amp-ad>
+```
+
+## Configuration
+
+For configuration details and to generate your tags, please contact us :- https://www.venatusmedia.com/contact or chrisb@venatusmedia.com
+
+Required parameter:
```
Comment (alanorozco) [inline ads/venatus.md]: Please pluralize ("parameters").

File: ads/venatus.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+<amp-ad width="300" height="250"
+	type="venatus"
+	data-sid="123456789"
+	data-pid="987654321">
+</amp-ad>
+```
+
+## Configuration
+
+For configuration details and to generate your tags, please contact us :- https://www.venatusmedia.com/contact or chrisb@venatusmedia.com
```
Comment (alanorozco) [inline ads/venatus.md]: Is there any way to avoid having this contact information here?

File: ads/venatus.js [hunk 5]
```diff
@@ -0,0 +1,27 @@
+/**
+ * Copyright 2016 The AMP HTML Authors. All Rights Reserved.
```
Comment (alanorozco) [inline ads/venatus.js]: Change to 2017 :)

**Comment by alanorozco** [comment]
I believe there is a problem with the CLA signature since some of the commits in this PR were authored by `cwbeck@gmail.com`.

**Comment by alanorozco** [comment]
@cwbeck 

Following up on this. Let me know if there's anything I can do to help you get this PR in.

**Comment by cwbeck** [comment]
@alanorozco - It's just the CLA issue. I've signed it with my personal email and as Venatus, not really much else I can do. I was hoping this would resolve before investing more time on this.

**Comment by alanorozco** [comment]
@cwbeck Thanks! CLA issue looks solved. Just a few of my comments to address before I can merge this.

**Pull request closed without merging**